tribuo-tutorials/enclave/src/main/resources/META-INF/native-image/serialization-config.json

```diff
@@ -0,0 +1,206 @@
+[
+  {
+    "name":"[[D"
+  },
+  {
+    "name":"[D"
```

Comment:
Why there is [D and [Lorg.tribuo.math.la.DenseVector ?

I saw this issue (got ObjectStreamClass with the name as "[D" and "[L...") when I de-serialize a KMeansModel.